### DISCUSSION TITLE
No icons in treeview

### DISCUSSION THREAD

**Discussion by xcariba** [eclipse-theia/theia#10980] [General]
### Bug Description:

There is no icons in explorer or other tree views.

![image](https://user-images.githubusercontent.com/7170514/161265172-700a2245-0803-4661-982d-0e268ef71346.png)

![image](https://user-images.githubusercontent.com/7170514/161265336-12663161-07af-4107-85d8-771cd39102e2.png)

### Steps to Reproduce:

1. Clone Theia master or pull changes
2. Call `yarn` in root and `yarn build` in `example/browser`
3. Start Theia `yarn start` in `example/browser`
4. Open any folder in Theia
5. Open Explorer, there will be no icons on file items in tree
6. Install any view that conrtibutes treeview, there will be no icons as well

### Additional Information

- Operating System: Fedora
- Theia Version: 1.24.0

**Comment by vince-fugnitto** [comment]
@xcariba did you set an `iconTheme`?

![image](https://user-images.githubusercontent.com/40359487/161267596-88e9ae71-1f0a-45e8-9cef-335d36c1286a.png)

https://github.com/eclipse-theia/theia/blob/96bee85f8464a60c96355de118d0757daaad20cc/dev-packages/application-package/src/application-props.ts#L66

**Comment by xcariba** [comment]
@vince-fugnitto I did not.
I've installed vs-seti icon theme and set it is preferences and now it shows icons in both explorer and my own plugin (that uses its own icons). When I set icon theme back to none icons disappeared in both explorer and my treeview vscode plugin.

**Comment by vince-fugnitto** [comment]
@xcariba that makes sense to me? 
The `none` icon theme sets the icons theme to `none`, meaning no icons in the individual nodes of the tree-view should exist.

**Comment by xcariba** [comment]
@vince-fugnitto maybe =)
I do expect that there will be no icons in default views that rely on icon theme, but my extension does not use them, and I need to install icon theme from vsix (like `vs-seti`) in order to set any icon theme so that my extension will show my own icons (from extension resources). Is there any option like `default` instead of `none`?
Also in this property description it says that it is file icon theme and my extension shows my own model tree, not a file tree, that is why it was unexpected (I think behavior changed a little bit due to #10938 ).

**Comment by vince-fugnitto** [comment]
@xcariba is your extension public where I can test it? The `none` icon-theme behaves as expected, where icons for views based on the file-tree should not display any icons (this behavior is also similar to vscode).

> Is there any option like default instead of none?

I think the only other builtin icon-theme is `theia-file-icons`:

https://github.com/eclipse-theia/theia/blob/96bee85f8464a60c96355de118d0757daaad20cc/packages/core/src/browser/icon-theme-contribution.ts#L46-L63

**Reply by xcariba** [reply]
@vince-fugnitto unfortunately it is not public, I'll try to find similar extension (I found Todo Tree extension, but it does not work in my Theia instance. There is task icon that is implemented similar to my case).
Thanks for `theia-file-icons`, it works for me!
